application/client/src/app/ui/service/toolbar.ts

```diff
@@ -137,6 +140,10 @@ export class Service extends Implementation {
         });
     }
 
+    public get chart(): Chart {
+        return this._chart;
+    }
+
```

Comment:
should not be here. Toolbar service is only for managing of toolbar: add/remove, active/deactive.